components/director/internal/domain/labeldef/service.go

```diff
@@ -60,3 +75,51 @@ func (s *service) List(ctx context.Context, tenant string) ([]model.LabelDefinit
 	}
 	return defs, nil
 }
+
+func (s *service) Update(ctx context.Context, def model.LabelDefinition) error {
+	if err := def.ValidateForUpdate(); err != nil {
+		return errors.Wrap(err, "while validating Label Definition")
+	}
+
+	if def.Schema == nil {
```

Comment:
let assume that I had definition with schema and then I decided to remove schema for it. In such case it won't be persisted in db. Write a test for such case.

Reply:
done